### DISCUSSION TITLE
azure-ad authentication for flexible postgres-server

### DISCUSSION THREAD

**Discussion by ridicolos** [dbeaver/dbeaver#19850] [Q&A]
Is there any way to configure DBeaver to use Azure AD Authentication for a PostgreSQL Database?
I only found a way to generate a access-token with `az` and then pass it to DBeaver as a password. But this would be a really bad user-experience for all our database-users. Is there any other way on how to realize this?

I found this, but it only seems to work with AzureSQL. https://github.com/dbeaver/dbeaver/wiki/Azure-Cloud-Explorer

**Comment by LonwoLonwo** [comment]
Hello @ridicolos 

You can try to use Azure AD in our Ultimate Edition version: https://github.com/dbeaver/dbeaver/wiki/Ultimate-Edition
